src/Frontend/src/resources/LicenseInfo.ts

```diff
@@ -7,6 +7,7 @@ export default interface LicenseInfo {
   instance_name: string;
   trial_license: boolean;
   license_status: LicenseStatus;
+  license_extension_url: string;
```

Comment:
it seems this new prop should be nullable?
```suggestion
  license_extension_url?: string;
```
which I assume will have repercussions in other areas of the code

Reply:
I might be missing something but the property doesn't need to be nullable. It has a well-defined default value which can be overridden by a value coming from the backend.

Reply:
The `LicenseInfo` type is retrieved by the call to `getLicense()` in the [serviceLicense](https://github.com/Particular/ServicePulse/pull/2108/files#diff-9aabc6274887bc0a98cc4eac00ef971b62366e71b4723aad50a4f14abcd391bbR147) file. This object is the evaluated in the `getOrUpdateLicenseStatus()` method, which is where the well-defined default value is provided.

So I think John is correct, the value here could be null.